.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
     "azureFunctions.showProjectWarning": false,
-    "editor.formatOnSave": true,
+    "editor.formatOnSave": false,
```

Comment:
I thought I commented on this earlier, but now I don't see it in the conversation history. Are we turning this off because it conflicts with another action (possibly the next line?)?

Reply:
Yeah eslint (the action from line 4) wants this space gone, but "formatOnSave" was adding it back.

<img width="583" alt="Screenshot 2024-02-05 at 2 29 09 PM" src="https://github.com/Azure/azure-functions-nodejs-e2e-tests/assets/11282622/8596b1da-3606-4e04-9cff-a52193b49073">

I can follow up separately from this PR to figure out ideal settings - I think the problem with "codeActionsOnSave" is that it only formats TypeScript files and it would be nice to format others files like json automatically as well. But suffice to say the conflict was super annoying lol